p/scripts/main.js

```diff
@@ -406,7 +429,6 @@ function last_feed() {
 
 function prev_category() {
 	const $active_cat = $('#aside_feed .tree-folder.active');
-
```

Comment:
I find a little whitespace easier to parse, personally. Whichever of course. :-)

Reply:
This section is not done yet. Still in jQuery 🤓